lib/MatchMediaController.ts

```diff
@@ -0,0 +1,49 @@
+import type { ReactiveControllerHost, ReactiveController } from 'lit';
+
+export class MatchMediaController implements ReactiveController {
+  /**
+   * The matchMedia result
+   * @see https://developer.mozilla.org/en-US/docs/Web/API/Window/matchMedia
+   */
+  public matches = false;
```

Comment:
yes! Way more sense to match the API.  love it

Reply:
Should be initialized to null, I think

Reply:
I reworked this a bit, removing the resize observer from MatchMediaController, for these reasons:
1) it's not called [ResizeController](https://github.com/lit/lit/blob/main/packages/labs/observers/src/resize_controller.ts)
2) ResizeObserver throws errors even when it works, and [that's not going to change](https://stackoverflow.com/questions/39354331/how-can-i-prevent-mocha-from-trapping-unhandled-exceptions). This interferes with testing. There are some [p. bad workarounds](https://stackoverflow.com/questions/39354331/how-can-i-prevent-mocha-from-trapping-unhandled-exceptions), but I figure, hey - our RO is really only just calling `matchMedia`, why not just listen for changes on the media query list instead?

Reply:
Yeah, makes sense. 👍